ArcGISRuntimeSDKQt_QMLSamples/Geometry/CutGeometry/CutGeometry.qml

```diff
@@ -132,6 +145,14 @@ Rectangle {
         cutButton.enabled = false;
     }
 
+    function resetPolygon() {
```

Comment:
Consider making this disabled by default, enabled after clicking cut, and then disabled again when clicked.